src/components/UpdateContract.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
I think there's a good reason to use this ESLint rules.
We'd want to have uniform `require`s.

Reply:
https://stackoverflow.com/questions/41292559/could-not-find-a-declaration-file-for-module-module-name-path-to-module-nam I faced that error while using an import statement so making this change fixed it.  I tried using it according to eslints rules but I faced the same error.

Reply:
Ah, go with that for now then. I'll fix it with a @types addition.